scripts/UsaData.py

```diff
@@ -10,84 +11,13 @@ def main():
     ## MODEL PARAMETERS
     ####################################
 
-    # Model Name 
-    model = ['Canada_USA']
-
-    # Regions in the model
-    regions = ['NAmerica']
-
-    # Years to loop over 
-    years = list(range(2019,2051,1))
-
-    # Emission Types
-    emissions = ['CO2']
-
-    # Storages
-    storages = [] 
-
-    # PWR Technologies
-    techsMaster = [
-        'BIO', # Biomass
-        'CCG', # Gas Combind Cycle 
-        'CTG', # Gas Combustion Turbine
-        'COA', # Coal
-        'COC', # Coal CCS
-        'HYD', # Hydro 
-        'SPV', # Solar
-        'URN', # Nuclear
-        'WND', # Wind
-    ]
-
-    #RWN Technologies
-    rnwTechs = [
-        'HYD', # Hydro 
-        'SPV', # Solar
-        'WND', # Wind
-        'BIO', # Biomass
-    ]
-
-    #MIN TEchnologies
-    mineTechs = [
-        'COA', # Coal
-        'GAS', # Gas
-        'URN', # Nuclear
-    ]
-
-    #STO Technologies
-    stoTechs = [
-    ]
-
-    #Countries and subregions in the model
-    countries = {
-        'USA':[] #Canada
-        }
-
-    #Read in subregions. This step is needed cause other scripts use the 
-    #provincial breakdown by subregion in the excel file 
-    sourceFile = '../dataSources/Regionalization.xlsx'
-    for country in countries:
-        dfCountry = pd.read_excel(sourceFile, sheet_name=country)
-        regionList = dfCountry['REGION'].tolist()
-        regionList = list(set(regionList)) # remove duplicates
-        countries[country] = regionList # save to dictionary
+    years, regions, emissions, techsMaster, rnwTechs, mineTechs, stoTechs, countries = functions.initializeCanadaUsaModelParameters('USA')
```

Comment:
Probably fine for this pr, but we should address with issue #70. Never been a huge fan of separating out USA and Canada, just to merge later on